.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "airbnb",
+
+    "rules": {
+        "indent": [2, 2],
```

Comment:
If 2-space is likely to get more contribution, then I'll bite my tongue. However it will be a bit inconsistent to have most of Wagtail's core JS on 4-space, but the new UI components 2-space. We should fix up the legacy code too, ideally.